lib/src/util/crypto_util.dart

```diff
@@ -82,7 +82,7 @@ UuidValue nameUuidFromBytes(List<int> name) {
   final bytes = md5.convert(name).bytes;
   bytes[6] = (bytes[6] & 0x0f) | 0x30;
   bytes[8] = (bytes[8] & 0x3f) | 0x80;
-  return UuidValue.fromList(bytes);
+  return UuidValue.fromList(Uint8List.fromList(bytes));
```

Comment:
the `bytes` from `md5.convert(name).bytes` should be `List<int>`
and  `UuidValue.fromList(List<int> byteList, {int? offset})` also accept `List<int>` as a parameter.

There should be no syntax errors here.

Reply:
whoops. I put the fix on the wrong line. Updated it.